samples/net/sockets/mt_http_server/README.rst

```diff
@@ -0,0 +1,62 @@
+.. _sockets-mt-http-server-sample:
+
+Socket Multithreaded HTTP Server
+################################
+
+Overview
+********
+
+The sockets/mt_http_server sample application for Zephyr implements a
+skeleton HTTP server using a BSD Sockets compatible API.
+This sample has similar functionality as :ref:`sockets-dumb-http-server-sample`
+except here we have support for multiple simultaneous connections and IPv6.
+Also the sample application has no compatibility with POSIX.
+This HTTP server example is very minimal and does not really parse an incoming
+HTTP request, just reads and discards it, and always serves a single static
+page.
+
+The source code for this sample application can be found at:
+:zephyr_file:`samples/net/sockets/mt_http_server`.
+
+Requirements
+************
+
+- :ref:`networking_with_host`
+- or, a board with hardware networking
+
+Building and Running
+********************
+
+Build the Zephyr version of the sockets/echo application like this:
```

Comment:
some cut-and-paste carry over here :)
Change to ``sockets/mt_http_server application``